region/maridia/inner-yellow/Bug Sand Hole.json

```diff
@@ -268,11 +269,20 @@
       "from": 3,
       "to": [
         {"id": 1},
-        {"id": 2}
+        {"id": 2},
+        {"id": 3}
       ]
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "Crystal Flash",
+      "requires": [
+        "canPrepareForNextRoom",
```

Comment:
Right, added requirements for that now. I always forget about the possibility that doors could lock behind you, since that doens't happen in Map Rando